### PR TITLE
Change default port number from 3030 to 4030.

### PR DESCRIPTION
**Pull request by pwendell**
Removes conflict with Typesafe's Zinc server default port.

https://spark-project.atlassian.net/browse/SPARK-901

### CODE REVIEW

File: ec2/spark_ec2.py [hunk 1]
```diff
@@ -215,11 +215,11 @@ def launch_cluster(conn, opts, cluster_name):
     master_group.authorize(src_group=slave_group)
     master_group.authorize('tcp', 22, 22, '0.0.0.0/0')
     master_group.authorize('tcp', 8080, 8081, '0.0.0.0/0')
-    master_group.authorize('tcp', 33000, 33000, '0.0.0.0/0')
+    master_group.authorize('tcp', 34040, 34040, '0.0.0.0/0')
     master_group.authorize('tcp', 50030, 50030, '0.0.0.0/0')
     master_group.authorize('tcp', 50070, 50070, '0.0.0.0/0')
     master_group.authorize('tcp', 60070, 60070, '0.0.0.0/0')
-    master_group.authorize('tcp', 3030, 3035, '0.0.0.0/0')
+    master_group.authorize('tcp', 4040, 3035, '0.0.0.0/0')
```
Comment (mateiz) [inline ec2/spark_ec2.py]: Hey Patrick, looks like you didn't update the range

File: ec2/spark_ec2.py [hunk 2]
```diff
@@ -215,11 +215,11 @@ def launch_cluster(conn, opts, cluster_name):
     master_group.authorize(src_group=slave_group)
     master_group.authorize('tcp', 22, 22, '0.0.0.0/0')
     master_group.authorize('tcp', 8080, 8081, '0.0.0.0/0')
-    master_group.authorize('tcp', 33000, 33000, '0.0.0.0/0')
+    master_group.authorize('tcp', 34040, 34040, '0.0.0.0/0')
```
Comment (mateiz) [inline ec2/spark_ec2.py]: This line actually isn't needed at all

**Comment by mateiz** [comment]
Saw a couple of issues in the EC2 script but go ahead and merge it once you fix those

**Comment by pwendell** [comment]
Thanks matei, I fixed those issues.

**Comment by mateiz** [comment]
Test

**Pull request merged**